testsuite/MathMLToDisplay/ErrorHandling/issue805.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
+  <head>
+    <title>Empty annotation element as first child of semantics</title>
+    <!-- Copyright (c) 2015 MathJax Consortium
+         License: Apache License 2.0 -->
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <script type="text/javascript" src="../../header.js"></script>
+    <script type="text/javascript">
+      function getTestCases() {
+        return [
+          newScriptReftestResult("No Math Processing Errors",true)
+        ];
+      }
+    </script>
```

Comment:
Since this is not a script retest any longer, you don't need this function (or the surrounding `<script>` tag).  It should no longer do anything anyway, since I think it is only called for if `scriptTests.js` is loaded.